src/webportal/src/app/job-submission/components/data/add-local.jsx

```diff
@@ -106,6 +124,17 @@ export const AddLocal = ({dataList, setDataList, setDataType}) => {
             ],
           }}
         />
+        {hdfsErrorMessage && (
+          <span
```

Comment:
Saw this many times, can we extra this?

Reply:
could we postpone the refactor design later because it may take some time and will change the codes a lot, though it is indeed reasonable.

Reply:
That's OK for me